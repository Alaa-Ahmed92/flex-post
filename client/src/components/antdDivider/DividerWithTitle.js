import React from 'react';
import { Divider } from 'antd';
import './style.css';

const DividerWithTitle = ({ title }) => {
    return (
        <Divider orientation="left" orientationMargin="0">
            {title}
        </Divider>
    )
};

export default DividerWithTitle;