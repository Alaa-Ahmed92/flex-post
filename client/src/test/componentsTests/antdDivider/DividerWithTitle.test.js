import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import DividerWithTitle from '../../../components/antdDivider/DividerWithTitle';

let props = {
    title: 'Comments'
};

const setUpComponent = () => {
    const component = shallow(<DividerWithTitle {...props} />);
    return component;
}

describe('DividerWithTitle component', () => {

    let component;
    beforeEach(() => {
        component = setUpComponent();
    });

    it('Should render the same with given props', () => {
        expect(shallowToJson(component)).toMatchSnapshot();
    });
});