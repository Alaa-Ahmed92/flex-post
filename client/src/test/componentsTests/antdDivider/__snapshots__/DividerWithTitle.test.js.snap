// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DividerWithTitle component Should render the same with given props 1`] = `
<Divider
  orientation="left"
  orientationMargin="0"
>
  Comments
</Divider>
`;
